### PR TITLE
Remove phrase trigger on post commit jobs

### PR DESCRIPTION
**Pull request by sijie**
Descriptions of the changes in this PR:

*Problem*

After enabling phrase trigger on precommit jobs to allow rerun individual precommit jobs, "retest this please" will run all the jobs including postcommit jobs.

*Solution*

Running postcommit jobs on pull requests doesn't make sense. This PR removes all the phrase trigger from postcommit jobs.

Additionally, fixes typos and also increases the timeout for integration tests.

### CODE REVIEW

**Comment by sijie** [comment]
@eolivelli yes it is unrelated. #1483 is fixing it.

File: bookkeeper-server/src/test/java/org/apache/bookkeeper/bookie/EntryLogTest.java [hunk 1]
```diff
@@ -1111,7 +1111,7 @@ public void testAppendLedgersMapOnCacheRemoval() throws Exception {
         EntryLogManagerForEntryLogPerLedger entryLogManager = (EntryLogManagerForEntryLogPerLedger) entryLogger
                 .getEntryLogManager();
 
-        long ledgerId = 0l;
+        long ledgerId = 0L;
```
Comment (jiazhai) [inline bookkeeper-server/src/test/java/org/apache/bookkeeper/bookie/EntryLogTest.java]: This seems already included in #1483

**Comment by sijie** [comment]
rerun pr validation

**Comment by sijie** [comment]
rebuild java9

**Comment by sijie** [comment]
The changes are mainly on jenkins DSL files. CI failures are not related. so ignore ci to allow merge.

**Comment by sijie** [comment]
IGNORE CI

**Pull request closed without merging**